pages/common/gofmt.md

```diff
@@ -0,0 +1,19 @@
+# gofmt
+
+> Gofmt formats Go programs. It uses tabs for indentation and blanks for alignment.
+
+- Format a file and print it to output:
+
+`gofmt {{source.go}}`
+
+- Format a file and rewrite original:
+
+`gofmt {{source.go}} -w`
+
+- Print all (including spurious) errors:
```

Comment:
Please add a `gofmt -s -w {{source.go}}` example above this. That is the most commonly used convention.